src/main/java/roomescape/reservation/ReservationController.java

```diff
@@ -0,0 +1,87 @@
+package roomescape.reservation;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import roomescape.reservation.dto.addReservationRequest;
+import roomescape.reservation.exception.MissingRequiredFieldException;
+import roomescape.reservation.exception.NotFoundReservationException;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class ReservationController {
+
+    private final List<Reservation> reservations = new ArrayList<>();
+
+    public ReservationController() {
+        reservations.add(new Reservation(1L, "브라운", "2023-01-01", "10:00"));
+        reservations.add(new Reservation(2L, "브라운", "2023-01-02", "11:00"));
+        reservations.add(new Reservation(3L, "브라운", "2023-01-03", "12:00"));
+    }
+
+    @GetMapping("/reservation")
+    public String readPage(Model model) {
+        model.addAttribute("reservation", reservations);
+        return "reservation";
+    }
+
+    @RequestMapping("/reservations")
```

Comment:
```suggestion
    @GetMapping("/reservations")
```

`@RequestMapping` 보다는, `@GetMapping` 어노테이션을 사용하는 것이 더 직관적일 것 같아요.

Reply:
`@RequestMapping`과 `@GetMapping`의 차이점에 대해 잘 몰랐는 데

1. `@RequestMapping`
모든 HTTP 메서드 지원, 특정 기능만을 지원하려면 추가 method 속성 설정필요

2. `@GetMapping`
GET 요청만 처리 지원

즉, `@GetMapping`이 `@RequestMapping`보다 좀 더 좁은 범위를 지원하는 군요!
어떤 역할을 하는 지 직관적으로 알려면 리뷰어는 말씀처럼 `@GetMapping`이 좀 더 적합할 거 같네요!!
짚어주셔서 감사합니다 :)